examples/golang/bucket-notification-deletion/notification-deletion-example/go.mod

```diff
@@ -0,0 +1,9 @@
+module notification-deletion/notification-deletion-example
+
+go 1.19
```

Comment:
why do we need go 1.19?
can we use go 1.18 (what i have on fedora36) or even go 1.17 (what i have on rhel8)?

Reply:
The new server I got is running go 1.19 that is why this is here.
I have changed it to go 1.18 as you advised.